lib/internal/errors.js

```diff
@@ -287,6 +287,8 @@ E('ERR_TRANSFORM_WITH_LENGTH_0',
 E('ERR_UNESCAPED_CHARACTERS',
   (name) => `${name} contains unescaped characters`);
 E('ERR_UNKNOWN_ENCODING', 'Unknown encoding: %s');
+E('ERR_UNKNOWN_FILE_EXTENSION', 'Uknown file extension: %s');
```

Comment:
typo: s/Uknown/Unknown

Reply:
Maybe this could print out the file too?

Reply:
Sure, instead of just outputting the extension we can output the full path.